src/saml-auth.md

```diff
@@ -15,15 +15,12 @@ management.
 Downloading the SAML authentication extension
 ---------------------------------------------
 
-The SAML authentication extension is available separately from the main
-`guacamole.war`. The link for this and all other officially-supported and
-compatible extensions for a particular version of Guacamole are provided on the
-release notes for that version. You can find the release notes for current
-versions of Guacamole here: <http://guacamole.apache.org/releases/>.
+```{include} include/sso-download.md
+```
 
-The SAML authentication extension is packaged as a `.tar.gz` file containing
-only the extension itself, `guacamole-auth-saml-1.4.0.jar`, which must
-ultimately be placed in `GUACAMOLE_HOME/extensions`.
+The extension for the desired SSO method, in this case
+`guacamole-auth-sso-openid-1.4.0.jar` from within the `openid/` subdirectory,
+must ultimately be placed in `GUACAMOLE_HOME/extensions`.
```

Comment:
Looks like a little copy-pasta, here - this should be "saml" and not "openid".

Reply:
Oopsies

Reply:
OK - fixed via rebase.